src/Sulu/Bundle/PageBundle/EventListener/WebspaceSerializeEventSubscriber.php

```diff
@@ -110,6 +110,7 @@ private function appendPortalInformation(Webspace $webspace, Context $context, S
      */
     private function appendUrls(Webspace $webspace, Context $context, SerializationVisitorInterface $visitor)
     {
+        return;
```

Comment:
Custom urls will no longer be part of the webspace serializer output

Reply:
They will still return urls but only the urls of the portal and no longer the list of all custom urls.

Reply:
@wachterjohannes Just out of curiosity how was this solved?